packages/package-json-lint/src/rules/require-ie10-compatible-dependency-versions.js

```diff
@@ -0,0 +1,41 @@
+const semver = require('semver');
+const maxSatisfying = require('semver/ranges/max-satisfying');
+const { execSync } = require('child_process');
+
+const versionSet = [
+  { name: 'redux', versionRange: '<4.0.0' },
+  { name: 'axios', versionRange: '<=0.18.1' },
+  { name: 'uuid', versionRange: '<7.0.0' },
+];
+
+const documentation = {
```

Comment:
Can we simplify the documentation just like what was done on https://github.com/cerner/terra-toolkit/pull/714?

Reply:
Updated here https://github.com/cerner/terra-toolkit/pull/713/commits/9b854148abd72f14d6c99fb8c1f20f55e2609e9a